lib/content-services/breadcrumb/breadcrumb.component.ts

```diff
@@ -95,13 +93,36 @@ export class BreadcrumbComponent implements OnInit, OnChanges {
             let node = this.transform ? this.transform(this.folderNode) : this.folderNode;
             this.route = this.parseRoute(node);
         }
+        this.recalculateNodes();
+    }
+
+    recalculateNodes(): void {
+        if (this.route.length > 3) {
```

Comment:
please don't add magic number in the code move it in a CONST that explain what is this number